recipes/pyvpt2/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "pyvpt2" %}
+{% set version = "0.0.1" %}
```

Comment:
I'd start with 0.1, as some machinery might only expect 0.0.* as a placeholder for on-the-fly computed versions